docs/configure.md

```diff
@@ -16,9 +16,42 @@ You do this by providing a `lqlint.json` file at the root of your project. Here'
     "rules": {}
 }
 ```
-
 This example has nothing switched on but is a good place to start; you can [grab it from here](../examples/lqlint.json).
 
+#### From the classpath
+
+It is also possible for Liquibase Linter to load the `lqlint.json` file from the classpath. This can be useful when you have many different
+projects using Liquibase and want to share the rule config between them. With maven this would be done in the following way, 
+where `lqlint.json` is stored directly under `src/main/resources` in `lqlint-config`
+
+```xml
+<plugin>
+    <groupId>org.liquibase</groupId>
+    <artifactId>liquibase-maven-plugin</artifactId>
+    <configuration>
+        <propertyFile>${liquibase.property.file}</propertyFile>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>com.whiteclarkegroup</groupId>
+            <artifactId>liquibase-linter</artifactId>
+            <version>${liquibaselinter.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.whiteclarkegroup</groupId>
+            <artifactId>lqlint-config</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+        </dependency>
+    </dependencies>
+    ...
+</plugin>
+```
+
+#### Custom lint config file path
```

Comment:
I think it would be good to provide an example here too. For example with Maven or Gradle how would you provide the property.

One good example use case might be if you distributed your config via an npm package, and so you'd want to provide a path like `node_modules/custom-lqlint-config/config.json` or something